olp-cpp-sdk-core/include/olp/core/logging/FilterGroup.h

```diff
@@ -150,9 +150,15 @@ inline boost::optional<Level> FilterGroup::getLevel() const {
   return m_defaultLevel;
 }
 
-inline void FilterGroup::setLevel(Level level) { m_defaultLevel = level; }
+inline FilterGroup& FilterGroup::setLevel(Level level) {
```

Comment:
Please move everything after here to FilterGroup.inl file

Reply:
Align members to coding style
m_defaultLevel -> default_level_
Can we remove boost::optional somehow? Do we need it here? Can we not just say default level is trace?